packages/thirdweb/src/chainsaw/urls.ts

```diff
@@ -0,0 +1,47 @@
+import { getThirdwebDomains } from "../utils/domains.js";
+
+export const getChainsawV2Url = (): string => {
+  const domain: string = getThirdwebDomains().chainsaw;
+  return `${domain}/v2`;
+};
+
+/**
+ * Endpoint to get block data
+ * @internal
+ */
+export const getBlockEndpoint = (blockNumber: number) =>
```

Comment:
prefer function

```suggestion
export function getBlockEndpoint(blockNumber: number) {
```

Reply:
Done in [8f6f7e4](https://github.com/thirdweb-dev/js/pull/3741/commits/8f6f7e4a4d9b73704a4bb032be1bb02c8d103181)